infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/VcsSshKeysProvisioner.java

```diff
@@ -137,7 +138,8 @@ private void doProvisionSshKey(SshPair sshPair, KubernetesEnvironment k8sEnv, St
         .getPodsData()
         .values()
         .forEach(
-            p -> mountSshKeySecret(secret.getMetadata().getName(), sshPair.getName(), p.getSpec()));
```

Comment:
Looks like tests do not cover it. Please consider improving tests

Reply:
done [here](https://github.com/eclipse/che/pull/14156/files#diff-7f761b9d2bffbe7b535b26f711aa892bR109)

Reply:
thanks for advice